.buildkite/premerge.steps.yaml

```diff
@@ -20,6 +20,7 @@ windows: &windows
     - "machine_type=quad"
     - "experiment_normalised_upload_paths=true"
     - "health_check_enabled=false"
+    - "maximum_instances=12"
```

Comment:
actually, shouldn't this be on the release qa pipeline too (for all PRs)

Reply:
Ah yeah you may be correct